README.md

```diff
@@ -343,17 +343,19 @@ For environment variables, each environment comprises a set of key value pairs d
     "local": {
         "version": "v2",
         "host": "localhost",
-        "token": "test token"
+        "token": "test token",
+        "secret": "devSecret"

```

Comment:
so consolidate with other references

Reply:
@Huachao - sorry I'm not quite following your comment.
Would it be worthwhile to merge this PR as is and raise new issue on document clarifications?

Reply:
Package-lock.json reverted